org.OpenT2T.Sample.SuperPopular.Helpers/js/index.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+var https = require('https');
+var q = require('q');
+
+//this is our base, we refactor these options in each method
+ var options = 
```

Comment:
The options object should not be global. It will cause problems when concurrent calls are trying to use the same object. Instead, create a new options object within the scope of each call. You can still define these strings separately as constants at the top if you want.